ui/model/game_history.py

```diff
@@ -71,10 +79,13 @@ def get_player_1_history(self):
         :return: a String
         """
         history_string = ""
-        player_1_history = self._history[::2]
-        for history_item in player_1_history:
-            history_string += str(history_item.move) + "\n"
-        print("Printing history 1 from game_history " + history_string)
+        player_1_history = self._history[0::2]
+        for i in range(1, len(player_1_history)):
+            history_string += \
+                str(i) + "." + "\n" + \
+                self.get_player_1_total_time(i-1) + " >>> " + \
+                self.get_player_1_total_time(i) + "\n" + \
+                str(player_1_history[i].move) + "\n"
```

Comment:
there's a lot of duplicate logic in this section -- could potentially refactor at some point into a single `get_player_history(color)` function or at least share some code behind the scenes